src/lib/dev-environment.js

```diff
@@ -99,6 +99,22 @@ export async function createEnvironment( slug, options ) {
 	await prepareLandoEnv( instanceData, instancePath );
 }
 
+export async function destroyEnvironment( slug ) {
+	debug( 'Will destroy an environment', slug );
+
+	const instancePath = getEnvironmentPath( slug );
+
+	debug( 'Instance path for', slug, 'is:', instancePath );
+
+	const environmentExists = fs.existsSync( instancePath );
+
+	if ( ! environmentExists ) {
+		throw new Error( 'Environment not found.' );
+	}
+
+	fs.rmdirSync( instancePath, { recursive: true } );
```

Comment:
I think we should probably run `lando destroy` here as well, to remove all the docker stuff that gets created by lando

Reply:
Great point! I added destroy.